opencodeblocks/scene/ipynb_conversion.py

```diff
@@ -0,0 +1,241 @@
+""" Module for converting ipynb data to ipyg data """
+
+from typing import Generator, OrderedDict, List, Dict
+
+MARGIN_X: float = 50
+MARGIN_BETWEEN_BLOCKS_X: float = 50
+MARGIN_Y: float = 50
+MARGIN_BETWEEN_BLOCKS_Y: float = 5
+BLOCK_MIN_WIDTH: float = 400
+TITLE_MAX_LENGTH: int = 60
+SOCKET_HEIGHT: float = 44.0
+TEXT_SIZE: float = 12
+TEXT_SIZE_TO_WIDTH_RATIO: float = 0.7
+TEXT_SIZE_TO_HEIGHT_RATIO: float = 1.42
+
+BLOCK_TYPE_TO_NAME: Dict[str, str] = {
+    "code": "OCBCodeBlock",
+    "markdown": "OCBMarkdownBlock",
+}
+
+
+def ipynb_to_ipyg(data: OrderedDict) -> OrderedDict:
+    """Convert ipynb data (ipynb file, as ordered dict) into ipyg data (ipyg, as ordered dict)"""
+
+    blocks: List[OrderedDict] = get_blocks(data)
+    edges: List[OrderedDict] = add_sockets(blocks)
```

Comment:
```suggestion
    edges_data: List[OrderedDict] = get_edges_data(data, blocks)
```

`add_sockets` should be named `get_edges_data` to match `get_blocks_data`
Also original data should be passed to `get_edges_data` in case edges data is present in .ipynb metadata